Core/src/main/java/com/plotsquared/core/util/PlayerManager.java

```diff
@@ -203,6 +209,57 @@ public static void getUUIDsFromString(
         return name;
     }
 
+    /**
+     * Attempts to resolve the username by an uuid
+     * <p>
+     * <b>Note:</b> blocks the thread until the name was resolved or failed
+     *
+     * @param owner The UUID of the owner
+     * @return A caption containing either the name, {@code None}, {@code Everyone} or {@code Unknown}
+     * @see #resolveName(UUID, boolean)
+     * @since 6.2.3
```

Comment:
That'd be 6.3.0 then, considering this is hardly a patch for 6.2. But you can also use `@since TODO`, I have a script that injects the parsed version from `build.gradle.kts` if I publish releases.